config/locales/alchemy.en.yml

```diff
@@ -57,6 +57,90 @@ en:
       main_menu: Main Menu
       footer_menu: Footer Menu
 
+    # == Mime Types translations
+    # These are human readable mime types used for the document-type row in archive files.
+    #
+    mime_types: &mime_types
```

Comment:
@tvdeyen I am utilising yml placeholders to be able to use the `mime_types` in the Attachment filter values. the placeholder must be defined before its used, so I had to move it up.